src/io/flutter/run/SdkRunConfig.java

```diff
@@ -142,8 +142,6 @@ public LaunchState getState(@NotNull Executor executor, @NotNull ExecutionEnviro
     final RunMode mode = RunMode.fromEnv(env);
     final Module module = ModuleUtilCore.findModuleForFile(mainFile.getFile(), env.getProject());
     final LaunchState.CreateAppCallback createAppCallback = (device) -> {
-      if (device == null) return null;
```

Comment:
Does removing this check cause any problems for regular flutter apps?
An alternate that might be more robust is to create a mock ChromeBrowser Device to use for FlutterWeb.